src/ui/Views/Shared/_Layout_base.cshtml

```diff
@@ -11,19 +11,15 @@
 
     <link rel="stylesheet" href="~/application.css" asp-append-version="true" />
 
-    <!--[if lt IE 9]>
```

Comment:
Do we not need the IE shiv? @dankmitchell

Reply:
That isn't working at present so best to remove. @the-daniel-rothig is going to create a card to add it properly.